### DISCUSSION TITLE
Anyone attempted a heterogeneous sized colored image Dataset like FGVCAircraft

### DISCUSSION THREAD

**Discussion by ankitgooner** [mrdbourke/pytorch-deep-learning#823] [Q&A]
HI all,

Please let me know if anyone tried creating a model for such datasets.I am facing some issues in reducing loss of my CNN which is way off. 

The issue i am facing in this case is since the output_features(the classes of aircrafts are close to 100).
Model takes a lot of time to run(15 mins so far with 3 epochs) and the results are still not accurate.

I tried everything from tweaking the data to change other model parameters.

**Comment by LuluW8071** [comment]
For such large dataset you may need to limit dataset to 25 labels or introduce random crops image augmentation
And maybe increase `conv2d` layers and add `dropout` with `ReLU` layers for `VGG model architecture`

**Comment by ankitgooner** [comment]
i tried with reducing number of output features as well. Getting some bizzare results
<img width="734" alt="image" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/23497810/eba81546-af91-402d-bcd4-994a6c93055e">

**Reply by LuluW8071** [reply]
Looks like your model isn't learning anything since test loss and accuracy is same
